### ISSUE TITLE
User Can't download without Access

### ISSUE THREAD

**Issue by davinderesfera** [danmunn/redmine_dmsf#599]
Hello 

I want one this on this Plugin, User has the ability to review the files only. If he/she wants to delete that file then they ask for the permission from admin. Is this Possible in this plugin.

Regards
Himanhsu

**Comment by picman** [comment]
Yes, just remove _Delete documents_ permission from all roles in the Redmine administration (Roles and permissions).

**Comment by davinderesfera** [comment]
Thanks for your reply I want no one can download the file without administrator permission.

**Comment by davinderesfera** [comment]
Actually mistakenly I wrote the "Delete" word. Actually, I Need to write "Download".

**Comment by picman** [comment]
There's no such permission, I'm afraid.

**Comment by mickeydonkey** [comment]
I think uploader of file should set visiblility of that file to a specific project member.
